gcloud-java-compute/src/main/java/com/google/gcloud/compute/StandardDiskConfiguration.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.compute;
+
+import com.google.api.services.compute.model.Disk;
+
+import java.util.Objects;
+
+/**
+ * A Google Compute Engine standard persistent disk configuration. This class allows users to create
+ * a disk given its type and size.
+ *
+ * @see <a href="https://cloud.google.com/compute/docs/disks/">Block Storage</a>
+ */
+public class StandardDiskConfiguration extends DiskConfiguration {
+
+  private static final long serialVersionUID = -6974045909359567054L;
+
+  /**
+   * A builder for {@code StandardDiskConfiguration} objects.
+   */
+  public static class Builder
+      extends DiskConfiguration.Builder<StandardDiskConfiguration, Builder> {
+
+    private Builder() {
+      super(Type.STANDARD);
+    }
+
+    private Builder(StandardDiskConfiguration configuration) {
+      super(configuration);
+    }
+
+    private Builder(Disk diskPb) {
+      super(Type.STANDARD, diskPb);
+    }
+
+    /**
+     * Creates a {@code StandardDiskConfiguration} object.
+     */
+    @Override
+    public StandardDiskConfiguration build() {
+      return new StandardDiskConfiguration(this);
+    }
+  }
+
+  private StandardDiskConfiguration(Builder builder) {
+    super(builder);
```

Comment:
If type is not provided `pd-standard` is used
If size is not provided 500GB is used
I will update the docs to add this information.